Homework 20

Apoorva0103 opened the pull request:
remove the thread.sleep from all the java classes and previous homework.
and updated to expilicit waits

Apoorva0103 committed:
Homework-19

Apoorva0103 committed:
Homework 20- with error

Apoorva0103 committed:
homework -20

remove the thread.sleep from all the java classes and previous homework.
and updated to expilicit waits

khanhtestpro commented:
//Comment from HW team,
Hello,
Your homework has been reviewed.
Home work 20 is empty please check
https://share.zight.com/X6ujP6dX

Best wishes from the HW team (Khanh, khanh@testpro.io)

Apoorva0103 commented:
i made the homework 20  java file, but homework was to change all the previous homework and change deletePlaylist() so i changed all those files and added in homework20 .

Apoorva0103 committed:
Update Homework20.java

khanhtestpro commented:
//Comment from HW team,
Hello,
Your homework has been reviewed.

I need you to create homework 20 so that the grader can easily distinguish between your remaining papers. Please check 
You are providing incorrect PR link.
Here is an example of a correct pull request. 
Plz resubmit with correct link.
Example (DO NOT USE):https://github.com/[your-github-username)/A40-QA-Automation-Framework/pull/1

Best wishes from the HW team (Khanh, [khanh@testpro.io](mailto:khanh@testpro.io))
